src/Dibi/Drivers/FirebirdDriver.php

```diff
@@ -324,7 +324,8 @@ public function escapeDateTime($value)
 	 */
 	public function escapeLike($value, $pos)
 	{
-		throw new Dibi\NotImplementedException;
+		$value = addcslashes(str_replace('\\', '\\\\', $value), "\x00\n\r\\'%_");
```

Comment:
Myslím že Firebird neescapuje lomítka. Tohle je fakt ožehavá operace a nemůžeš kopírovat kód od jinud pokud si nejsi 1000% jistý, co dělá…

Reply:
Možná by mohlo fungovat

```php
		$value = strtr($value, ["'" => "''", '%' => '[%]', '_' => '[_]', '[' => '[[]']);
		return ($pos <= 0 ? "'%" : "'") . $value . ($pos >= 0 ? "%'" : "'") . "ESCAPE ''''";
```

Ale nemám možnost jak to ověřit, a pokud neoveříš, že to skutečně funguje jak má, tak se mi to moc nechce mergovat…